nidaqmx/task.py

```diff
@@ -2,6 +2,8 @@
 import numpy
 import six
 import warnings
+import nidaqmx._grpc_stub_interpreter as _grpc_stub_interpreter
```

Comment:
This eagerly imports gRPC modules.

nimi-python lazily imports gRPC modules, so that non-gRPC applications will still work without installing `grpcio`:
https://github.com/ni/nimi-python/blob/master/generated/nidcpower/nidcpower/session.py#L6866

```
        if grpc_options:
            import nidcpower._grpc_stub_interpreter as _grpc_stub_interpreter
            interpreter = _grpc_stub_interpreter.GrpcStubInterpreter(grpc_options)
        else:
            interpreter = _library_interpreter.LibraryInterpreter(encoding='windows-1251')
```

It doesn't declare a hard dependency on `grpcio`: https://github.com/ni/nimi-python/blob/master/generated/nidcpower/setup.py